rts/Sim/Misc/ModInfo.cpp

```diff
@@ -336,10 +335,16 @@ void CModInfo::Init(const std::string& modFileName)
 			throw content_error("Sensors\\Los\\AirLosMipLevel out of bounds. The minimum value is 0. The maximum value is 30.");
 	}
 	{
-		//misc
-		const LuaTable& misc = root.SubTable("misc");
+		// root level (with fallbacks in subtables)
 
-		windChangeReportPeriod = static_cast<int>(math::roundf(misc.GetFloat("windChangeReportPeriod", static_cast<float>(windChangeReportPeriod) / GAME_SPEED) * GAME_SPEED));
+		const float wcrpSeconds = root.GetFloat(     "windChangeReportPeriod"
+		                        , root.GetFloat("misc.windChangeReportPeriod"
+		                        , windChangeReportPeriod * INV_GAME_SPEED));
+		windChangeReportPeriod = static_cast<int>(math::roundf(wcrpSeconds * GAME_SPEED));
+
+		insertBuiltUnitMoveCommand = root.GetBool(             "insertBuiltUnitMoveCommand"
```

Comment:
That's because `insertBuiltUnitMoveCommand` was never in the `construction` subtable from a stable releases PoV, it was put there like 10 commits ago and no games use it. This means we can put it anywhere. I wouldn't have put it into the `construction` subtable in the first place but I was asked to keep it there.